### PR TITLE
Feature/QPPBSR-8035: PY2021 Y5 WI benchmarks

### PR DESCRIPTION
**Pull request by lavioli-work**
#### Motivation for change

Y5 benchmark updates for WI

#### What is being changed

Add the measures required for WI in wi-benchmark.json and add measures to the 2021.json in the benchmark directory.

Reference: https://confluence.cms.gov/display/QPPBSR/Y5+WI+Benchmark+Updates

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [ ] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [x] Documentation updated
* [ ] Unit tests added/passing
* [ ] Verified working locally

##### Associated JIRA tickets:
https://jira.cms.gov/browse/QPPBSR-8035

### CODE REVIEW

**Review by fhuang-semanticbits — APPROVED** [review]

**Review by okeefem3 — COMMENTED** [review]
LGTM, could we get a veteran of this repo like @bartcone-sb, @KyleApfel, @pgrocholski or @saquino0827 to just double check and make sure we aren't doing anything weird?

**Comment by okeefem3** [comment]
We documented the process for future developers https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPBSR&title=Y5+WI+Benchmark+Updates

File: staging/2021/benchmarks/json/wi-benchmarks.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
+[
+  {
+    "measureId": "110",
+    "benchmarkYear": 2021,
```
Comment (okeefem3) [inline staging/2021/benchmarks/json/wi-benchmarks.json:4]: @lavioli-work I believe we need to make all of the `benchMark` years `2019` across the board, could you make this change?

**Review by okeefem3 — APPROVED** [review]

**Review by KyleApfel — APPROVED** [review]
image

Comparing against the QPP Resource library benchmarks I found nothing out of order. All looks good here!

**Pull request merged**